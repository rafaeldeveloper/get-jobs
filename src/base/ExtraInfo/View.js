import React from 'react'

export default ({ data }) => (
    <>
        <span>
            Local : {data?.place || "Não Informado"}
        </span><br />
        <span>
            Salario: {data?.salary || "Não Informado"}
        </span>
    </>
)