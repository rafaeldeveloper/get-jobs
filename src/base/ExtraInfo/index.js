import Loading from './View'
export default Loading