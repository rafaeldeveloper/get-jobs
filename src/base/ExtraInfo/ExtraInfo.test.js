import React from 'react';
import ExtraInfo from 'base/ExtraInfo';
import renderer from 'react-test-renderer';




test('ExtraInfo renders correctly', () => {

    const component = renderer.create(
        <ExtraInfo/>
    );

    let tree = component.toJSON();
    
    expect(tree).toMatchSnapshot();
''
});

