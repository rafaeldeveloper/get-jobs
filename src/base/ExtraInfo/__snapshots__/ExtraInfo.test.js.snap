// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExtraInfo renders correctly 1`] = `
Array [
  <span>
    Local : 
    Não Informado
  </span>,
  <br />,
  <span>
    Salario: 
    Não Informado
  </span>,
]
`;
